patternfly/patternfly-elements
# Updates to pfelement supporting new compilation approach

## Updates to pfelement supporting new compilation approach

* The pfelement css assets were not being moved to the dist directory due to the updated compilation tasks, this is being fixed in this branch.

---

### What has changed and why
Summarize files edited as part of this MR along with a brief description of what was changed/why.

* Renamed pfelement-noscript.css to pfelement--noscript.css
* Added pfelement.css to the assets list in the pfelement object in the package.json
* Updated references to and documentation of the --noscript.css file

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1. Run `npm run build pfelement`, confirm that pfelement.css (minified versions and maps) and pfelement--noscript.css (same) are available in the dist directory.

Your repository infrastructure updates should work for at least:

### Ready-for-merge Checklist



**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
